From 0bf338f94cd2d75d334f9a17eb149c63eb434fbc Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 3 May 2023 14:24:17 +0200
Subject: [PATCH 0184/1141] arm64: defconfig: Enable Moortec DTS sensor support

This patch adds in built-in the Moortec DTS sensor driver
used by STM32MP SOCs.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: Ic77b2b635953cb45cd7ecb0ee0dfb95945de3744
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305107
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index d0d46b1517fd..805e4f00de73 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -19,6 +19,7 @@ CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
+CONFIG_SENSORS_MR75203=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
 CONFIG_SPI_STM32=m
-- 
2.39.2

